Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
```

Comment:
Oh, I have no objection on logic here. But I do have a more general question: should we support rebottling a poured keg at all? To me, it feels a bad use case, considering bottles are likely useless.